src/frontend/components/check-list/check-list.tsx

```diff
@@ -0,0 +1,98 @@
+/*
+ * This file is part of NER's PM Dashboard and licensed under GNU AGPLv3.
+ * See the LICENSE file in the repository root folder for details.
+ */
+
+import { ReactNode, useState } from 'react';
+import PageBlock from '../../layouts/page-block/page-block';
+import { Form, Button, Modal } from 'react-bootstrap';
+import styles from './check-list.module.css';
+
+interface CheckListProps {
+  title: string;
+  headerRight?: ReactNode;
+  list: {
+    details: string;
+    resolved: boolean;
+  }[];
+}
+
+const CheckList: React.FC<CheckListProps> = ({ title, headerRight, list }) => {
+  const [checks, setChecks] = useState(list);
+  const [show, setShow] = useState(false);
+
+  const handleClose = () => setShow(false);
+  const handleShow = () => setShow(true);
+
+  const handleCheck = (idx: number) => {
+    const updatedChecks = checks.map((check, i) => {
+      if (i === idx) {
+        check.resolved = !check.resolved;
+        return check;
+      }
+      return check;
+    });
+
+    setChecks(updatedChecks);
+  };
+
+  const builtList = checks.map((check, idx) => (
+    <div key={idx} className={styles.container}>
+      <Form.Check
+        label={
+          <p
+            style={check.resolved ? { textDecoration: 'line-through' } : { textDecoration: 'none' }}
+          >
+            {check.details}
+          </p>
+        }
+        defaultChecked={check.resolved}
+        data-testId={`testCheckbox${idx}`}
+        onChange={() => handleCheck(idx)}
+      />
+      {check.resolved ? (
+        <Button variant="danger">Delete</Button>
+      ) : (
+        <Button variant="success">Convert to CR</Button>
+      )}
+    </div>
+  ));
+
+  builtList.push(
+    <div>
```

Comment:
So for this whole section it looks like you're taking in checks that you've declared elsewhere and then using it to render all these react components. I would reformat this so that bulletlist is kinda a custom bit of React in this file. I believe
`const BulletList = (checkList: any) => { return ()}` should work and then you put all the code for the map inside a div inside the return statement (maybe wrap that in a div) and I think that'll work and then you should be able to just add the Add New Risk button below it in the top div rather than appending it to the list of jsx components as you've done here. 

I also suggest this method because it allows you to call bulletlist below like `<BulletList checkList = {checks} />` rather than `{builtList}`; I think this looks a lil cleaner/ makes it more clear what bulletlist is/ what it takes in.

Let me know if this suggestion is confusing btw, I know it's a lot. It's a lil easier to show in code, so I can hop in a call or smthn if need be.